src/backend/modules/url/url.service.js

```diff
@@ -18,10 +18,11 @@ class UrlServiceImp {
 
     async createOne({ url, slug }, userDetail, urlRecaptcha) {
         let newSlug;
-        fetch(urlRecaptcha, {
+        await axios({
             method: 'post',
+            url: urlRecaptcha
         })
-            .then(response => response.json())
+            .then(response => response.data)
             .then(async googleResponse => {
                 if (googleResponse.success === true) {
                     const keywords = parseUrl(url);
```

Comment:
??? Why do you combine await with .then ??? It doesn't work like that
https://javascript.info/async-await